sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -37,6 +37,28 @@ import { UserAgentOptions } from '@azure/core-http';
 import { UserDelegationKeyModel } from '@azure/storage-blob';
 import { WebResource } from '@azure/core-http';
 
+// @public
+export interface AccessControlChangeCounters {
+    changedDirectoriesCount: number;
+    changedFilesCount: number;
+    failedChangesCount: number;
+}
+
+// @public
+export interface AccessControlChangeFailure {
+    errorMessage: string;
```

Comment:
Should this be an Error not a Failure (and thus have `name` and `message` like typical errors)?

Reply:
`Failure` is initially from storage service concept about a failed action for a ACL change. I think it's good to be an `Error` in JS world.

Reply:
Updated to an Error with name and message properties.